README.md

```diff
@@ -11,22 +9,52 @@
+- **500+ Listed Cryptocurrencies:** A vast and ever-growing list of supported cryptocurrencies.
+- **Unlimited markets:** Over 300,000 completed atomic swaps and a staggering 10,000+ trading pairs.
+- **User support:** Engage with our active Discord support channel, where official team members are ready to help. We'd also love to hear your feedback.
+- **Most supported protocols:** AtomicDEX outshines the competition by supporting the most blockchain protocols of any decentralized exchange on the market.
+- **Established team:** Our pioneering Komodo developers have been at the forefront of DEX technology since 2014.
+- **Frequent updates:** Our dedicated developers are constantly working to enhance the app's user experience, security, and feature set.
+
+## Download AtomicDEX Mobile Wallet 📲
+Embrace financial freedom at your fingertips by downloading the AtomicDEX Mobile Wallet:
+
+- [iOS](https://testflight.apple.com/join/c2mOLEoC)
+- [Android](https://play.google.com/store/apps/details?id=com.komodoplatform.atomicdex)
+
+## Get Involved 🤝
+We welcome developers, designers, and testers' contributions to our open-source project. If you'd like to contribute, please review the [contribution guidelines](CONTRIBUTING.md) and [code of conduct](CODE_OF_CONDUCT.md).
+
+For any questions about the AtomicDEX Mobile Wallet or the exchange, join our lively [Discord Support channel](https://komodoplatform.com/discord). Official team members are identifiable by the "Komodo Team" role.
 
 # NB (Forkers/contributors):
-This repository is currently in the procesundergoing safety and Flutter version upgrade. Expect major merge conflicts in the near future for any files updated from this repository. Please note that this software is under active development and is provided "as is" without any warranties or guarantees. Use at your own risk, as the authors and maintainers shall not be held liable for any issues, damages, or losses resulting from its use.
+
+This repository is currently in the process of undergoing safety and Flutter version upgrades. Expect major merge conflicts in the near future for any files updated from this repository. Please note that this software is under active development and provided "as is" without warranties or guarantees. Use at your own risk, as the authors and maintainers shall not be held liable for any issues, damages, or losses resulting from its use.
 
 ## Getting Started
 
 Build requires up-to-date version of coins file from https://github.com/KomodoPlatform/coins
 
 Commit hash and sah256sum of coins file is specified in `coins_ci.json`.
-You may download one manually or use `fetch_coins.sh` script on linux and macOS,
- `fetch_coins.ps1` powershell script on Windows.
+You may download one manually or use `fetch_coins.sh` script on Linux and macOS,
+ `fetch_coins.ps1` PowerShell script on Windows.
 
 `fetch_coins` script depends on sha256sum and jq utils:
```

Comment:
~`fetch_coins` script depends on sha256sum and jq utils:~
`The `fetch_coins` script depends on sha256sum and jq utils:`